assets/zip_asset_store.cc

```diff
@@ -43,8 +43,10 @@ std::unique_ptr<fml::Mapping> ZipAssetStore::GetAsMapping(
     const std::string& asset_name) const {
   TRACE_EVENT1("flutter", "ZipAssetStore::GetAsMapping", "name",
                asset_name.c_str());
-  auto found = stat_cache_.find(asset_name);
 
+  std::stringstream file_name;
+  file_name << directory_.c_str() << "/" << asset_name;
+  auto found = stat_cache_.find(file_name.str());
```

Comment:
nit: I think `auto found = stat_cache_.find(directory_ + "/" + asset_name);` might be cleaner.

Reply:
Done